android-buddy-plugin/src/main/java/com/likethesalad/android/buddy/configuration/AndroidBuddyPluginConfiguration.kt

```diff
@@ -18,7 +20,18 @@ class AndroidBuddyPluginConfiguration
         librariesScopeMapper.librariesScopeExtensionToLibrariesScope(extension.librariesPolicy.scope)
     }
 
+    private val transformationScope by lazy { extension.transformationScope }
+
     fun getLibrariesScope(): LibrariesScope {
         return lazyLibrariesScope
     }
+
+    fun getTransformationScope(): MutableSet<in QualifiedContent.Scope> {
+        val scopes = transformationScope.scope.getOrElse(mutableSetOf(QualifiedContent.Scope.PROJECT.name))
+        return scopes.map { scopeName -> QualifiedContent.Scope.valueOf(scopeName) }.toImmutableSet()
```

Comment:
I would like to avoid as much as possible to use Google's or any other dependency's classes as part of the plugin configuration, such as this `QualifiedContent.Scope` enum for example. This is because if they change in future versions (which has already happened for said enum), then the users of `android-buddy` would have to change their build.gradle files in order to accommodate to an android-buddy's internal tool change which can be quite annoying for them.

I'd rather create our own enum with the options that we want for `android-buddy` to have, which I believe can be only 2:
- PROJECT
- DEPENDENCIES

And then in this part of the code we would "translate" those into the `QualifiedContent.Scope` equivalent items, so for our enum's "PROJECT" we'd add "QualifiedContent.Scope.PROJECT" to the list, and for our enum's "DEPENDENCIES" we'd add both "QualifiedContent.Scope.SUB_PROJECTS" and "QualifiedContent.Scope.EXTERNAL_LIBRARIES" to it.

Reply:
agree, will change this.

Reply:
@LikeTheSalad I fixed this with the latest PR update